README.md

```diff
@@ -18,18 +18,24 @@ Clone the repository in your computer:
 $ git clone git@github.com:adriacidre/ali.git
 ```
 
-Then set up your ./basrc or ./zshrc with some commands like the ones below with your own $GOPATH
+Then set up your ./zshrc with the following command:
+
+```
+$ make install
+```
+
+And set up the ali with some commands like the ones below with your own $GOPATH
 
 ```
-[ -f ~/.aliases ] && source ~/.aliases
 ali() {
-    $GOPATH/github.com/adriacidre/ali/ali $@
-    source ~/.aliases
-    if [[ "$1" == "rm" ]] then
-        unalias $2
-    fi
+    $GOPATH/github.com/adriacidre/ali/ali $@	
+    source ~/.aliases	
+    if [[ "$1" == "rm" ]] then	
+        unalias $2	
```

Comment:
review your IDE as it's adding trailing spaces on each line

Reply:
I'll remove the spaces